tests/helper_functions.py

```diff
@@ -154,8 +154,7 @@ def pyfunc_serve_and_score_model(
                                              declaring the scoring process to have failed.
     :param extra_args: A list of extra arguments to pass to the pyfunc scoring server command. For
                        example, passing ``extra_args=["--env-manager", "local"]`` will pass the
-                       ``--env-manager local`` flag to the scoring server to ensure that conda
-                       environment activation is skipped.
+                       ``--env-manager local`` flag to the scoring server to ensure that conda environment activation is skipped.
```

Comment:
Nice! Let's fix `.github/workflows/matchers/pylint.json` first and make sure the warning will turn into an error.